pixeldata/src/lib.rs

```diff
@@ -833,13 +836,26 @@ impl DecodedPixelData<'_> {
                             }
                             (VoiLutOption::Default | VoiLutOption::First, None) => {
                                 tracing::warn!("Could not find window level for object");
-                                Lut::new_rescale_and_normalize(
-                                    8,
-                                    signed,
-                                    rescale,
-                                    data.iter().copied(),
-                                )
-                                .context(CreateLutSnafu)?
+                                let voi_lut_sequence = self.voi_lut_sequence.clone();
+
+                                if voi_lut_sequence.is_some() {
+                                    let voi_lut_sequence = voi_lut_sequence.unwrap();
```

Comment:
rather than check + unwrap, why not just `if let Some(voi_lut_sequence) = voi_lut_sequence)`?